internal/server/v0/triple/golden/get_triples/observation.json

```diff
@@ -101,46 +101,5 @@
       "objectName": "Total Population",
       "provenanceId": "dc/base/WorldDevelopmentIndicators"
     }
-  ],
-  "dc/o/w2z8nx9y43k97": [
-    {
-      "subjectId": "dc/o/w2z8nx9y43k97",
-      "predicate": "observationAbout",
-      "objectId": "Earth",
-      "objectName": "World",
-      "provenanceId": "dc/base/WorldDevelopmentIndicators"
-    },
-    {
-      "subjectId": "dc/o/w2z8nx9y43k97",
-      "predicate": "observationDate",
-      "objectValue": "2006",
```

Comment:
@n-h-diaz This value is present for 2006:
https://screenshot.googleplex.com/BUKK9wJhuaWn4kr

Reply:
Is this deletion okay?

Reply:
@n-h-diaz what was the resolution here?

Reply:
This golden file is from BigQuery and this observation misses data there. See the following query:

SELECT * FROM `datcom-store.dc_kg_latest.StatVarObservation` 
WHERE id = "dc/o/w2z8nx9y43k97" or id = "dc/o/88cs3xqnmpp55"

vs

SELECT * FROM `datcom-store.dc_kg_2023_12_08_23_23_15.StatVarObservation` 
WHERE id = "dc/o/w2z8nx9y43k97" or id = "dc/o/88cs3xqnmpp55"

Reply:
Based on the team chat, the resolution seems to be to update the golden file which is now done. @pradh please confirm that this is what you intended? @ajaits @shifucun @n-h-diaz FYI

Reply:
Yes, please update the golden file with `dc/o/em7gh269k7s2`

Reply:
hmm, the test file seems updated, but not seeing additions in this file?

Reply:
Updated!